src/gui/generalsettings.cpp

```diff
@@ -660,4 +659,19 @@ void GeneralSettings::slotRemotePollIntervalChanged(int seconds)
     cfgFile.setRemotePollInterval(interval);
 }
 
+void GeneralSettings::updatePollIntervalVisibility() {
+    const auto accounts = AccountManager::instance()->accounts();
+    const auto pushAvailable = std::any_of(accounts.cbegin(), accounts.cend(),
+    [](const AccountStatePtr &accountState) -> bool {
+        if (!accountState)
+            return false;
+        AccountPtr accountPtr = accountState->account();
+        if (!accountPtr)
+            return false;
+        return accountPtr->capabilities().availablePushNotifications().testFlag(PushNotificationType::Files);
+    });
```

Comment:
nitpick: formatting and const auto

```suggestion
    const auto pushAvailable = std::any_of(accounts.cbegin(), accounts.cend(), [](const AccountStatePtr &accountState) -> bool {
        if (!accountState) {
            return false;
        }
        const auto accountPtr = accountState->account();
        if (!accountPtr) {
            return false;
        }
        return accountPtr->capabilities().availablePushNotifications().testFlag(PushNotificationType::Files);
    });
```